pioinstaller/penv.py

```
@@ -16,18 +16,153 @@
+            if file in ("uv", "uv.exe"):
+                return os.path.join(root, file)
+    return None
+
+
+def download_and_install_uv(cache_dir):
+    """Download and install uv package manager."""
+    uv_platform = get_uv_platform()
+    if not uv_platform:
+        raise exception.PIOInstallerException(
+            f"Unsupported OS/architecture for uv: {platform.system()}/{platform.machine()}"
+        )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)